neuroinfer/code/run_bayesian.py

```diff
@@ -228,15 +303,38 @@ def run_bayesian_analysis_area(
 
     # Initialize an empty list to store results and coordinates
     result_all = []
-    coord = []
+    # Define padding
+    padding = int(radius / 3)
+
+    # Initialize visited_coord array
+    visited_coord = np.zeros(
+        (
+            np.max(coordinates_area[0]) - np.min(coordinates_area[0]) + 1,
+            np.max(coordinates_area[1]) - np.min(coordinates_area[1]) + 1,
+            np.max(coordinates_area[2]) - np.min(coordinates_area[2]) + 1,
+        )
+    )
+    coord_ranges = {
```

Comment:
at this point coordinates_area is a N list of triplets. coordinates_area[0] contains the first triplets of coordinates

Reply:
we might want something like:
"xmax" : max([a[0] for a in coordinates_area]),